GitUI/CommandsDialogs/FormEditor.cs

```diff
@@ -42,6 +42,7 @@ public FormEditor(GitUICommands commands, string? fileName, bool showWarning, bo
             panelMessage.Visible = showWarning;
 
             fileViewer.IsReadOnly = readOnly;
+            fileViewer.InitalLineNumber = lineNumber;
```

Comment:
Why not just forward `lineNumber` to `OpenFile`?

Reply:
The text is loaded in ShowDialog(), calling SetText(), so it cannot be set yet.